### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by Julien-Molina**

### CODE REVIEW

File: Examples/build-osx.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (maurelio1234) [reply Examples/build-osx.sh]: use a single script with a parameter?

File: Examples/xc.authentication/build-osx.sh [hunk 2]
```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+##########################################################################
+# This is the Cake bootstrapper script for Linux and OS X.
```
Comment (maurelio1234) [inline Examples/xc.authentication/build-osx.sh]: >  This is the Cake bootstrapper script for Linux and OS X.

why do we need two scripts?

File: Examples/build-linux.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+succeededBuilds=()
+failedBuilds=()
+
+for d in $(find . -maxdepth 1 -type d | grep 'xc*')
+do
+    #Do something, the directory is accessible with $d:
```
[the remark this answers is not in the document]
Comment (maurelio1234) [reply Examples/build-linux.sh]: a less verbose way:

```
for b in $(find . | grep build-linux.sh)
do
    cd $(dirname $b)
    ./build-linux.sh
    ...
    cd -
done

```

File: Examples/xc.authentication/build-osx.sh (hunk 2 above)
Comment (Julien-Molina) [reply Examples/xc.authentication/build-osx.sh]: I don't know, I just took it as it is from koordinator

Comment (Julien-Molina) [reply Examples/xc.authentication/build-osx.sh]: Cake provides a script for Linux and a script for OS X. But the comment in the header is the same.

**Pull request merged**